modules/installation-restricted-network-samples.adoc

```diff
@@ -26,23 +42,32 @@ not addressed in this procedure.
 
 .Procedure
 
+. Access the images of a specific imagestream to mirror, for example:
++
+----
+$ oc get is <imagestream> -n openshift -o json | jq .spec.tags[].from.name | grep registry.redhat.io
+----
++
 . Mirror images from link:registry.redhat.io[registry.redhat.io] associated with any imagestreams you need
-in the restricted network environment into one of the defined mirrors:
+ifdef::restrictednetwork[]
+in the restricted network environment into one of the defined mirrors, for example:
+endif::[]
+ifdef::configsamplesoperator[]
+into your defined preferred registry, for example:
+endif::[]
 +
 ----
-$ oc image mirror myregistry.com/myimage:latest myregistry.com/myimage:stable
+$ oc image mirror registry.redhat.io/rhscl/ruby-25-rhel7:latest ${MIRROR_ADDR}/rhscl/ruby-25-rhel7:latest
 ----
 +
 . Add the required trusted CAs for the mirror in the cluster’s image
 configuration object:
 +
 ----
-$ oc create configmap registry-config --from-file=$path/ca.crt -n openshift-config
+$ oc create configmap registry-config --from-file={MIRROR_ADDR}=$path/ca.crt -n openshift-config
```

Comment:
@bmcelvee My apologize, I should inform you more clear in the early comment.
MIRROR_ADDR is composed by the registry hostname and port 5000, such as qeci-2783.mirror-registry.qe.gcp.devcluster.openshift.com:5000 ,
 then `oc create configmap registry-config --from-file={MIRROR_ADDR}=$path/ca.crt -n openshift-config` should be  like this `oc create configmap registry-config --from-file=qeci-2783.mirror-registry.qe.gcp.devcluster.openshift.com..5000=$path/ca.crt -n openshift-config`